modules/gcloud/bigtable.go

```diff
@@ -7,12 +7,13 @@ import (
 	"github.com/testcontainers/testcontainers-go/wait"
 )
 
-// Deprecated: use RunBigTable instead
+// Deprecated: use [RunBigTable] instead
```

Comment:
question: this looks like a deprecation chain, could this point to the ultimate desired function?

Reply:
Yeah, I was thinking about that, if pointing to the new function or keep it as is. I'll directly link to the new ones, everywhere